### PR TITLE
Merge 177x into 178x

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | Merge 177x into 178x
| Type?             | improvement
| Category?         | ME
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | N/A
| How to test?      | N/A

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/25451)

### CODE REVIEW

**Review by matks — DISMISSED** [review]
Merge content seems OK, no unwanted changes. But CI is not happy with this state 😅

**Review by matks — DISMISSED** [review]

File: src/Adapter/Product/AdminProductWrapper.php [hunk 1]
```diff
@@ -144,7 +144,7 @@ public function processProductAttribute($product, $combinationValues)
         // choice but to hard code this one to make sure enough precision is saved in the DB or it results in errors
         // of 1 cent in the shop
         $computingPrecision = CustomMoneyType::PRESTASHOP_DECIMALS;
-        if (!isset($combinationValues['attribute_ecotax'])) {
+        if (!isset($combinationValues['attribute_ecotax']) || 0 == (float) $combinationValues['attribute_ecotax']) {
```
Comment (eternoendless) [inline src/Adapter/Product/AdminProductWrapper.php]: Shouldn't this be...
```suggestion
        if (!isset($combinationValues['attribute_ecotax']) || 0.0 === (float) $combinationValues['attribute_ecotax']) {
```

Comment (Progi1984) [reply src/Adapter/Product/AdminProductWrapper.php]: More logical

**Review by eternoendless — APPROVED** [review]

**Review by matks — APPROVED** [review]

**Comment by matks** [comment]
Thank you @Progi1984

**Pull request merged**